runner/runners/combine.go

```diff
@@ -0,0 +1,44 @@
+package runners
```

Comment:
file name should be the same as the struct it implements.  This makes finding this when browsing code much easier.